src/Wallabag/CoreBundle/GuzzleSiteAuthenticator/GrabySiteConfigBuilder.php

```diff
@@ -47,6 +54,12 @@ public function buildForHost($host)
             $host = substr($host, 4);
         }
 
+        if (empty($this->credentials[$host])) {
```

Comment:
Of course https://github.com/wallabag/wallabag/pull/3101/files#diff-98420eb79a38a096ce347d5f58bf1b85R93